docs/tutorials/formula_interface/formula_interface.ipynb

```diff
@@ -0,0 +1,1724 @@
+{
+ "cells": [
```

Comment:
I agree with Matthias that the sneak peek would be better placed before the table of contents.

There is a warning that appears twice related to the intercept. If they are there on purpose (you want to show that a warning will appear), then do it just once. Otherwise, fix the warning.

Reply:
Ohh and also, don't refer to "tutorial 1", use a more explicit name to the tutorial you are referring to.